openassessment/xblock/static/sass/oa/utilities/_shame.scss

```diff
@@ -189,10 +189,6 @@
     }
   }
 
-  .ui-staff__title {
```

Comment:
Note to reviewers: this bit of CSS is unnecessary. The same thing is being set in _staff.scss (without the !important), and it works just fine with this removed. I verified that ui-staff__title is only used in the places edited above.